Modules/Microsoft365DSC/DSCResources/MSFT_AADNamedLocationPolicy/MSFT_AADNamedLocationPolicy.psm1

```diff
@@ -411,6 +413,7 @@ function Export-TargetResource
     )
     $ConnectionMode = New-M365DSCConnection -Workload 'MicrosoftGraph' -InboundParameters $PSBoundParameters
 
+    Select-MgProfile -Name 'Beta'
```

Comment:
Why are we forcing beta? The Get-MgIdentityConditionalAccessNamedLocation is available on the v1.0 endpoints. We should always try to prioritize using v1.0 over beta when available.

Reply:
Updated